### ISSUE TITLE
JavaScript string extraction and po_to_json conversion add extra backslashes into the catalogs

### ISSUE THREAD

**Issue by mzazrivec** [webhippie/gettext_i18n_rails_js#42]
Say you have a javascript file `app/test.js` with the following content:
```javascript
__('First "string"');
__("Second \"string\"");
```

After string extraction, you'll end up with this in the catalog:
```
msgid "First \"string\""
...
msgid "Second \\\"string\\\""
```

After `po_to_json`, you'll end up with the following in your `app.json`:
```
"First \\\"string\\\"":[""],"Second \\\\\\\"string\\\\\\\"":[""]
```

At this point the original strings in your `app/test.js` and the `app.json` catalog won't match, i.e. even though they were technically translated, the extraction and conversion made them into different strings.

**Comment by tboerger** [comment]
If you got a fix a pull request is welcome :)

**Comment by mzazrivec** [comment]
I was looking into this for a bit but it's not exactly easy to fix (not yet anyway).

**Comment by tboerger** [comment]
I think this will be fixed by https://github.com/webhippie/po_to_json/pull/12, right?

**Comment by chaomao** [comment]
> I think this will be fixed by [webhippie/po_to_json#12](https://github.com/webhippie/po_to_json/pull/12), right?

yes, @tboerger  this PR is trying to fix the issue :)
